### DISCUSSION TITLE
Can I utilize APIs on my own website?

### DISCUSSION THREAD

**Discussion by JordanIKlein** [community/community#28471] [Apps, API and Webhooks] [answered]
Hello there,
I have a question about utilizing GitHub APIs on my own website. I was thinking of adding fields (payload) to my site so users are able to interact with GitHub's API. They'd use their own Token of course, but I do have ads running on my site. I wanted to know if this is possible or would GitHub not allow it.
Thanks!

**Comment by M0nicaVaz** [comment, marked as answer]
Hey!
I couldn't find anything about ads, but this might help you:
https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#h-api-terms

**Reply by JordanIKlein** [reply]
Thank you so much!
